src/components/App/App.tsx

```diff
@@ -12,32 +12,76 @@ import React, { FC } from "react";
 import "./App.css";
 
 const App: FC = () => {
-    const { isDarkMode, updateName, name } = useSignatureContext();
+    const { isDarkMode, updateState, name, email, company, title, linkedin, phone } = useSignatureContext();
     const { profile } = useAuthenticationContext();
 
     return (
-        <ProviderV3 theme={isDarkMode ? darkTheme : lightTheme} colorScheme={"light"} height="100%">
-            <ToastContainer />
-            <Layout>
-                <Flex marginX="size-300" alignItems="start" direction="column" justifyContent="center" gap={"size-200"}>
-                    {profile ? <h3>Welcome {profile?.given_name}</h3> : <></>}
-                    <UsernameInput
-                        label={"Name"}
-                        text={name.textValue}
-                        isBold={name.isBold}
-                        isItalics={name.isItalics}
-                        onInputChange={updateName}
-                    />
-                    {/* <UsernameInput label={"Title"} />
-                    <UsernameInput label={"Company"} />
-                    <UsernameInput label={"Email"} />
-                    <UsernameInput label={"Phone number"} />
-                    <UsernameInput label={"LinkedIn"} /> */}
-                    <ImgInput />
-                    <TablePreview />
-                </Flex>
-            </Layout>
-        </ProviderV3>
+        <AuthenticationProvider>
```

Comment:
We can remove this AuthenticationProvider wrapping tag as it's being used in the index.tsx (the root of the project) thus removing BUG 🪲 🔥